components/SwipeListView.js

```diff
@@ -38,15 +38,26 @@ class SwipeListView extends Component {
 		if (this.props.closeOnRowBeginSwipe && this.openCellId && this.openCellId !== id) {
 			this.safeCloseOpenRow();
 		}
+
+    if (this.props.swipeGestureBegan) {
```

Comment:
Looks like the formatting got messed up here and everywhere else. Could you make sure you're using tabbed indentation and just see if the formatting matches the rest of the file, thanks!